src/gui/editors/PianoRoll.cpp

```diff
@@ -1985,6 +2033,7 @@ void PianoRoll::mouseReleaseEvent( QMouseEvent * me )
 				m_pattern->instrumentTrack()->pianoModel()->
 						handleKeyRelease( note->key() );
 				note->setIsPlaying( false );
```

Comment:
I'd prefer to have all `Note::setIsPlaying` after all chord notes are paused, for consistency, and also to avoid race conditions in the future.

Reply:
You're right